meeteval/io/__main__.py

```diff
@@ -0,0 +1,159 @@
+                            f'input file and "-" is required to pipe out: '
+                            f'"cat file.stm | meeteval-io stm2rtm - | ..."',
+                formatter_class=SmartFormatter,
+                add_help=False,
+            )
+            command_parser.add_argument(
+                '--help', 
+                help='Show this help message and exit',
+                action='help',
+                default=argparse.SUPPRESS,
+            )
+            command_parser.add_argument(
+                '--force', '-f',
+                help='Overwrite the output file if it exists.',
+                action='store_true',
+                default=False,
+            )
+            command_parser.add_argument(
+                'input_files', 
+                nargs='+' if not piping_in else '*',
+                help='The input files.',
+            )
+            command_parser.add_argument(
+                'output_file', type=str,
+                help='The output file. "-" means stdout.'
+            )
+            command_parser.set_defaults(
+                input_format=reader,
+                output_format=writer,
+                parser=command_parser,
+            )
+
+            # Special cases
+            if reader == 'ctm':
+                command_parser.add_argument(
+                    '--speaker',
+                    type=str,
+                    help='The speaker name to use for the CTM. Anything in '
+                         'curly braces is replaced by the segment information, '
+                         'for example "--speaker \'{session_id}\'". '
+                         'Defaults to the file stem.',
+                    required=piping_in,
+                    default=None if piping_in else '{filestem}',
+                )
+            if reader == 'rttm' and writer in ('stm', 'seglst'):
+                command_parser.add_argument(
+                    '--words',
+                    help='Placeholder for missing words. Anything in curly '
+                         'braces is replaced by the segment information, '
+                         'for example "{session_id}-{speaker}".',
+                    default='<NA>',
+                )
+
+    args = dict(vars(parser.parse_args()))
+    if not args['force']:
+        if os.path.exists(args['output_file']):
+            args['parser'].error(
+                f'Output file "{args["output_file"]}" already exists. '
+                'Use --force / -f to overwrite.'
+            )
+    if piping_in:
+        if args['input_files']:
+            import subprocess
+            args['parser'].error(
+                f'Input files ({subprocess.list2cmdline(args["input_files"])}) '
+                f'are not allowed when piping into the command.'
+            )
+        args['input_files'] = [sys.stdin]
+    if args['output_file'] == '-':
+        args['output_file'] = sys.stdout
```

Comment:
Can dump handle the '-'?

Reply:
Yes, after adding '-' handling to `_open`